frontend/src/App.js

```diff
@@ -8,6 +8,7 @@ import Result from './pages/result'
 import EntriesList from './pages/entries'
 import Topics from './pages/topics'
 import Navbar from './components/Navbar/Navbar'
+import About from './pages/about'
```

Comment:
ich würde die pages dann auch mit großem Anfangsbuchstaben schreiben also hier About

Reply:
meinst du damit, die url? die habe ich absichtlich komplett in Kleinschreibung gehalten.